src/LayoutManager/LayoutStrategies/LayoutStrategy.ts

```diff
@@ -89,19 +88,13 @@ export abstract class LayoutStrategy {
         size: bboxSize,
         center: bboxCenter,
       });
-      const originFactor = new Point(
-        -resolveOrigin(target.originX),
-        -resolveOrigin(target.originY)
-      );
-      const sizeCorrection = actualSize
-        .subtract(bboxSize)
-        .multiply(originFactor);
-      // translate the layout origin from left top to target's origin
-      const center = bboxLeftTop.add(bboxSize.multiply(originFactor));
```

Comment:
Seems to me this is accounted for here:
https://github.com/fabricjs/fabric.js/blob/0547c05dbb68ebe8437f6bb9783f393d463d0721/src/LayoutManager/LayoutManager.ts#L207
and some line lower for the case of non initialization